File: Uri-iniciante/1037.c

#include<stdio.h>
/*	
*	-----------------------------------
*	|		Pedro Daniel Jardim       |
*	| 			  UFV				  | 
*	|		  27/02/2020			  |
*   ----------------------------------
*
*/



int main() 
{

 float a = 0;
scanf("%f", &a);

if (a >= 0 && a <= 25)
  printf("Intervalo [0,25]\n");
else if (a > 25 && a <= 50)
  printf("Intervalo (25,50]\n");
else if (a > 50 && a <= 75)
  printf("Intervalo (50,75]\n");
else if (a > 75 && a <= 100)
  printf("Intervalo (75,100]\n");
else
  printf("Fora de intervalo\n");




   return 0;
   
}

File: Beginner/1037.py
value = float(input())

if 0 <= value <= 25.0000:
    print("Intervalo [0,25] ")
elif 25.0001 <= value <= 50.0000:
    print("Intervalo (25,50]")
elif 50.0001 <= value <= 75.0000:
    print("Intervalo (50,75]")
elif 75.0001 <= value <= 100.000:
    print("Intervalo (75,100]")
elif value < 0 or value > 100:
    print("Fora de intervalo")